src/core/pointcloud/qgspointcloudrenderer.h

```diff
@@ -673,6 +798,11 @@ class CORE_EXPORT QgsPointCloudRenderer
     Qgis::PointCloudSymbol mPointSymbol = Qgis::PointCloudSymbol::Square;
     int mPainterPenWidth = 1;
     Qgis::PointCloudDrawOrder mDrawOrder2d = Qgis::PointCloudDrawOrder::Default;
+
+    bool mRenderAsTriangles = false;
+    bool mHorizontalTriangleFilter = false;
+    float mHorizontalTriangleFilterThreshold = 5.0;
```

Comment:
I think this should be double -- the internal calculations can use float, but it's weird for user-exposed values to eg change from 1.20 to 1.1999999 in the GUI.

Reply:
Done